backend/recipes/tasks.py

```diff
@@ -0,0 +1,165 @@
+from recipes import celery
+import hardware
+
+
+def heat(parameters):
+    """
+    Turn on the heater and reach a target temperature.
+
+    :param parameters:
+        dictionary
+            'temp' - the desired temperature
+    :return:
+        None
+    """
+    targetTemp = parameters['temp']
+    celery.logger.info('heating water to ' + str(targetTemp) + '...')
```

Comment:
Based on pep standards we should format the string like

`"My name is {0}".format('Fred')`

https://peps.python.org/pep-3101/